scripts/broken-links-slack-payload.mjs

```diff
@@ -0,0 +1,72 @@
+/* eslint-disable no-console */
+import fs from 'fs'
+import core from '@actions/core'
+import { program } from 'commander'
+
+const DEFAULT_INPUT_FILE = 'broken-links-report.json'
+const DEFAULT_OUTPUT_FILE = 'broken-links-slack-payload.json'
+const GROUP_TO_NOTIFY = 'subteam^S010U41C30V|cms-helpdesk' // '<!subteam^S010U41C30V|cms-helpdesk>'
```

Comment:
This should be changed back to the commented version once we discuss this with CMS Helpdesk. The current version does not @ them.